fractal_server/app/runner/_process/__init__.py

```diff
@@ -173,3 +59,29 @@ async def process_workflow(
     output_task_pars = output_task_pars_fut.result()
     output_dataset_metadata = output_task_pars.metadata
     return output_dataset_metadata
+
+
+async def process_workflow(
+    *,
+    workflow: Workflow,
+    input_paths: List[Path],
+    output_path: Path,
+    input_metadata: Dict[str, Any],
+    logger_name: str,
+    workflow_dir: Path,
+    username: str = None,
+    worker_init: Optional[
```

Comment:
We'll need to look back at this also in the SLURM backend, since we may want to include additional lines for a task. Nothing urgent, let's do this once we have relevant use cases.